docs/docs/api/rest/workflows/README.md

```diff
@@ -0,0 +1,247 @@
+
+The `template_id` for your workflow can be found from the URL of the shared workflow link you created in **Step 1** of this guide.
+
+The `trigger` as a `url` prop, so let's provide it with a specific URL (`https://hnrss.org/newest?q=Pipedream`) for this new workflow:
+
+```json{7-11}
+// Example of a Create Workflow request payload
+{
+  "project_id": "proj_abc123",
+  "org_id": "o_abc123",
+  "template_id": "tch_abc123",
+  "triggers": [
+    {
+      "props": {
+        "url": "https://hnrss.org/newest?q=Pipedream"
+      }
+    }
+  ]
+}
+```
+
+:::tip Triggers are addressable by index
+
+You may have noticed that we didn't include the `namespace` argument to the trigger in our payload. This is because triggers are ordered sequentially, whereas steps need a `namespace` argument for proper addressing.
+
+:::
+
+If we were to send this payload to the **Create Workflow** endpoint now, it will populate the *RSS - New Item in Feed* trigger with the feed we provided.
+
+You can also populate the `steps` props as well. 
+
+The *Slack - Send message in a Public Channel* step requires a `channelId`, `message` and the connected Slack account (`slack`). Let's start with connecting the Slack account.
+
+#### Find your connected accounts
+
+To connect your accounts to the workflow, you'll need to find the specific IDs for each of the accounts you'd like to connect.
+
+You can find your connected account IDs by using the [List Accounts endpoint](/api/rest/#get-workspaces-s-connected-accounts).
+
+You can filter your accounts by using the `app_slug` query parameter. For example, if you want to find your connected Slack accounts to your workspace, then add `slack` to the query param:
+
+```
+GET /workspaces/<workspace_id>/accounts?app_slug=slack
+```
+
+This request will narrow down the results to only your connected Slack accounts, for easier finding.
+
+You'll need the ID of each connected account you'd like to configure this new workflow with. Copy down the `apn_******` value of the connected accounts from the response you'd like to use for the steps.
+
+```json{10}
+{
+  "page_info": {
+    "total_count": 1,
+    "count": 1,
+    "start_cursor": "YXBuXzJrVmhMUg",
+    "end_cursor": "YXBuXzJrVmhMUg"
+  },
+  "data": [
+    {
+      "id": "apn_abc123",
+      "name": "Slack Pipedream Workspace"
+    }
+  ]
+}
+```
+
+Now we can copy the ID for our Slack account from the response: `apn_abc123`.
+
+Given we now have the connected account ID, we can design the rest of the payload:
+
+```json{13-20}
+{
+  "project_id": "proj_abc123",
+  "org_id": "o_abc123",
+  "template_id": "tch_3BXfWO",
+  "triggers": [
+    {
+      "props": {
+        "url": "https://hnrss.org/newest?q=Pipedream"
+      }
+    }
+  ],
+  "steps": {
+    {
+      "namespace": "send-message",
+      "props": {
+        "slack": "apn_abc123",
```

Comment:
Currently, I think the app's configured prop needs to be an object with key `authProvisionId`.

```suggestion
        "slack": {
          "authProvisionId": "apn_abc123",
        }
```